phpunit/bootstrap.php

```diff
@@ -134,6 +134,46 @@ function gutenberg_register_test_block_for_feature_selectors() {
 			),
 		)
 	);
+
+	WP_Block_Type_Registry::get_instance()->register(
+		'my/block-with-selectors',
```

Comment:
One note here, the previous test block named `test/test` contains several `__experimentalSelector` keys. Should we remove that part, update with a comment, update the existing tests?

Reply:
We need to maintain backward compatibility for the `__experimentalSelector` properties. Also, if a block has the `selectors` property that will be used over any legacy `__experimentalSelector` so it seemed a little cleaner to me to keep those test blocks separate.